### PR TITLE
foooorkkkk

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
seem ok?

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "create-react-context",
+  "name": "@4c/create-context",
```
Comment (taion) [inline package.json:2]: `4c/create-react-context`?

**Review by taion — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jquense) [reply package.json:2]: it felt redundant :P

**Comment by jquense** [comment]
meh lets jsut use the original I think the API is back to jsx

**Pull request closed without merging**